test/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+    "compilerOptions": {
+        "esModuleInterop": true
```

Comment:
Why do you set that flag ?

For ``mock-fs`` ?

Reply:
That's correct, mock-fs uses commonjs.